llvm/test/Instrumentation/MemorySanitizer/or.ll

```diff
@@ -1,4 +1,4 @@
-; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; NOTE: Assertions have mostly been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
which part is not autogenerated?

Reply:
The auto-generator will duplicate lines between CHECK-PRECISE and CHECK-IMPRECISE, instead of putting common lines into CHECK.